pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -399,6 +428,17 @@ func ValidateLimitedPriorityLevelConfiguration(lplc *flowcontrol.LimitedPriority
 	return allErrs
 }
 
+func ValidateExemptPriorityLevelConfiguration(eplc *flowcontrol.ExemptPriorityLevelConfiguration, fldPath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+	if eplc.NominalConcurrencyShares < 0 {
```

Comment:
once the optional field is a pointer, this will need a nil check.

Reply:
yes

Reply:
Done in the force-push to 90d57e6a7e7